test-case-reporting/README.md

```diff
@@ -1,31 +1,51 @@
 AMP Test Case Reporting Bot
 ==============
 
-[[description]]
+An Google Cloud app that stores the results of tests run by Travis. Used to track flakey tests.
 
 This app runs as a [[something]].
 
 Interface
 ---------
-[[what this listens for, what it does]]
+### API for Travis
+
+The App has the following API points, which are to be triggered from Travis CI
+runs.
+
+* `POST /report`
+  * Accepts a JSON object representing a test result report from Travis. Has 3 fields: `build`, `job`, and `results`, representing build information, job information, and test run information respectively. For the full structure of the object, see [the type declaration module](types/test-case-reporting.d.ts).
 
 Setup
 -----
-[[no clue]]
+
+Follow these setup instructions to start developing for this App locally:
+
+1. Clone this repository and cd into the `test-case-reporting` directory
+2. `npm install`
+3. Create a new Google Cloud project and switch to it with `gcloud config set project [PROJECT_NAME].
```

Comment:
```suggestion
3. Create a new Google Cloud project and switch to it with `gcloud config set project [PROJECT_NAME].`
```

Reply:
Applied manually to avoid including the period at the end of the sentence.